app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -234,6 +235,16 @@ class HomeActivityTest {
 
   @Test
   fun testHomeActivity_recentlyPlayedStoriesTextIsDisplayed() {
+    storyProgressTestHelper.markRecentlyPlayedForFractionsStory0Exploration0(
+      profileId = ProfileId.newBuilder().setInternalId(internalProfileId1).build(),
+      timestampOlderThanAWeek = false
+    )
+    testCoroutineDispatchers.runCurrent()
+    storyProgressTestHelper.markRecentlyPlayedForRatiosStory0Exploration0(
+      profileId = ProfileId.newBuilder().setInternalId(internalProfileId1).build(),
+      timestampOlderThanAWeek = false
+    )
+    testCoroutineDispatchers.runCurrent()
```

Comment:
Also, we are using `testCoroutineDispatchers.runCurrent()` alot, once the test passes correctly we can check if we need these or not.